src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -1536,6 +1536,7 @@ else if (SlimefunManager.isItemSimiliar(current, SlimefunItems.PULVERIZED_ORE, t
 			@Override
 			public boolean onBlockBreak(BlockBreakEvent e, ItemStack item, int fortune, List<ItemStack> drops) {
 				if (SlimefunManager.isItemSimiliar(item, SlimefunItems.AUTO_SMELT_PICKAXE, true)) {
+					if (BlockStorage.check(e.getBlock()) != null) return false;
```

Comment:
You should use `if (BlockStorage.hasBlockInfo(e.getBlock())) return false;`. It's what the method is for.